app/views/admin/loan_questions/_form.html.slim

```diff
@@ -18,6 +18,9 @@
     = f.input :status, label: t("loan_questions.active"), as: :boolean, checked_value: 'active',
       unchecked_value: 'inactive'
 
+    = f.input :division_id, collection: current_user.accessible_divisions,
+      selected: @loan_question.division.id
```

Comment:
This can be `division_id` instead of `division.id` -- a little more efficient since a database call won't be needed.